audiophiler/__init__.py

```diff
@@ -63,62 +63,126 @@
 @app.route("/")
 @auth.oidc_auth('default')
 @audiophiler_auth
-def home(auth_dict=None):
+def home_def(auth_dict=None):
+    return redirect("/1")
+
+@app.route("/<int:page>")
+@auth.oidc_auth('default')
+@audiophiler_auth
+def home(page, auth_dict=None):
     # Retrieve list of files for templating
-    db_files = File.query.all()
+    query = File.query
+    rows = query.count()
+    rows = int(rows // 10 + bool(rows % 10))
+
+    if page > rows or page < 1:
+        return "Page Out of Bounds", 404
+
+    db_files = File.query.offset((page-1) * 10).limit(10).all()
     harolds = get_harold_list(auth_dict["uid"])
     tour_harolds = get_harold_list("root")
+
     is_rtp = ldap_is_rtp(auth_dict["uid"])
     is_eboard = ldap_is_eboard(auth_dict["uid"])
+
     return render_template("main.html", db_files=db_files,
                 get_date_modified=get_date_modified, s3_bucket=s3_bucket,
                 auth_dict=auth_dict, harolds=harolds, tour_harolds=tour_harolds,
-                is_rtp=is_rtp, is_eboard=is_eboard, is_tour_page=False)
+                is_rtp=is_rtp, is_eboard=is_eboard, is_tour_page=False,
+                current="", page=page, rows=rows, begin=max(1, page-6),
+                end=min(page+6, rows) + 1)
```

Comment:
same comment about begin and end as your other pagination pr. can this be done in the jinja since you're passing page?